platform/darwin/src/MGLOverlay.h

```diff
@@ -30,10 +30,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  The approximate center point of the overlay area. (required) (read-only)
```

Comment:
Isn’t this misleading too? For now, just say that it’s a coordinate representing the overlay.